apps/plugin-browser/src/scripts/background.ts

```diff
@@ -65,6 +65,11 @@ browser.tabs.onUpdated.addListener((tabId, changeInfo) => {
           const applicableRules = automaticInferenceConfig.rules.filter(
             ({ restrictToDomains }) => {
               const pageHostname = new URL(pageDetails.pageUrl).hostname;
+
+              if (pageHostname === "app.hash.ai") {
```

Comment:
@CiaranMn Can we also add `hash.ai` to this? We'd like to move the site over ASAP, so let's just cover both bases and do it all here now.